internal/dag/builder.go

```diff
@@ -527,9 +528,15 @@ func (b *Builder) computeRoutes(sw *ObjectStatusWriter, proxy *projcontour.HTTPP
 
 	visited = append(visited, proxy)
 	var routes []*Route
+
+	// Check for duplicate conditions on the includes
+	if includeConditionsIdentical(proxy.Spec.Includes) {
+		sw.SetInvalid("duplicate conditions defined on an include is invalid")
```

Comment:
Nit: since we are setting the status to invalid, can we remove that from the reason and just say: "duplicate include conditions" or "duplicate conditions defined on an include"?

Reply:
:+1: